code/modules/mob/living/simple_animal/guardian/guardian.dm

```diff
@@ -72,18 +72,18 @@
 			loc = get_turf(summoner)
 
 
-/mob/living/simple_animal/hostile/guardian/adjustBruteLoss(amount) //The spirit is invincible, but passes on damage to the summoner
+/mob/living/simple_animal/hostile/guardian/adjustCloneLoss(amount) //The spirit is invincible, but passes on damage to the summoner
```

Comment:
All damage to simple animals coverts to typeless health loss regardless, so yeah it doesn't really matter beyond the elusive "in code lore"